components/editor/modules/link/ui.js

```diff
@@ -120,6 +121,24 @@ export default ({TYPE}) => {
         )
       )
     }
+    const repoChange = (onChange, value, node) => repo => {
+      onChange(
+        value.change().replaceNodeByKey(
+          node.key,
+          {
+            type: TYPE,
+            kind: 'inline',
+            data: node.data.merge({
+              title: repo.text,
```

Comment:
looks wrong, probably repo.value.meta.title or something

Reply:
Nope. You get a RepoSearch result here and the text always equals the repo title.